helm/charts/infra/values.yaml

```diff
@@ -120,7 +120,9 @@ server:
   #   fsGroup: 2000
 
   ## Security context for the server container
-  securityContext: {}
+  securityContext: {
+    readOnlyRootFilesystem: true
```

Comment:
Enabling `readOnlyRootFilesystem` by default. This will help us not break "read-only" in the future accidentally.

Reply:
This sets the security context for the server container, not the pod. For that you'll need to set `podSecurityContext`. It also doesn't set the security context for the connector

Reply:
Oops my bad. `readOnlyRootFilesystem` is only applicable for `securityContext`, not `podSecurityContext`

Reply:
The only case where this would be a problem is if the user is using sqlite _and_ isn't using a PVC, i.e. `server.persistence.enabled=false`. That use case should be extremely rare since it recreates the database on pod restart